examples/hits_example.py

```diff
@@ -0,0 +1,81 @@
+import os
+from typing import Any
+
+import graphblas as gb
+import graphblas_algorithms as ga
+
+import numpy as np
+import scipy.sparse as sps
+from numpy.testing import assert_allclose
+
+os.environ["SPARSE_BACKEND"] = "Finch"
+import sparse
+
+# select namespace
+xp = sparse  # np jnp
+Array = Any
+
+
+def converged(xprev: Array, x: Array, N: int, tol: float) -> bool:
+    err = xp.sum(xp.abs(x - xprev))
+    return err < xp.asarray(N * tol)
+
+
+class Graph:
+    def __init__(self, A: Array):
+        assert A.ndim == 2 and A.shape[0] == A.shape[1]
+        self.N = A.shape[0]
+        self.A = A
+
+
+@sparse.compiled()
+def kernel(hprev: Array, A: Array, N: int, tol: float) -> tuple[Array, Array, Array]:
+    a = hprev.mT @ A
+    h = A @ a.mT
+    h = h / xp.linalg.vector_norm(h, ord=xp.inf)
+    conv = converged(hprev, h, N, tol)
+    return h, a, conv
+
+
+def hits_finch(G: Graph, max_iter: int = 100, tol: float = 1e-8, normalized: bool = True) -> tuple[Array, Array]:
+    N = G.N
+    if N == 0:
+        return xp.asarray([]), xp.asarray([])
+
+    h = xp.full((N, 1), 1.0 / N)
+    A = xp.asarray(G.A)
+
+    for _ in range(max_iter):
+        hprev = h
+        a = hprev.mT @ A
+        h = A @ a.mT
+        h = h / xp.linalg.vector_norm(h, ord=xp.inf)
```

Comment:
```suggestion
        h = h / xp.max(h, axis=...)
```

Why obfuscate? 😉 `xp.max` is much simpler than `xp.linalg.vector_norm(..., ord=xp.inf)`.

Reply:
In this use case I wanted to show broader use of Array API, and show that `linalg` is also a standardized namespace, so it serves showcase purposes.
But I can change it to `xp.max`/`xp.sum` if you think it suits presentation better.

Reply:
I'd only use the linalg namespace when we have things that can't be expressed as functions in the main namespace.